notebooks/sprints/sprint19/530_rsclient_staging.ipynb

```diff
@@ -0,0 +1,520 @@
+      "                    'updated_at': '2024-12-16T14:14:58.068153'},\n",
+      "                {   'created_at': '2024-12-15T17:44:37.842562',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': '519e6d47-d65e-4b20-93c7-19a3484fe3b2',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-15T17:44:42.927917'},\n",
+      "                {   'created_at': '2024-12-15T18:07:35.902912',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': '50d7708a-49c4-4e6a-b136-562b15df4dfe',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-15T18:07:43.426583'},\n",
+      "                {   'created_at': '2024-12-17T13:38:42.281778',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': 'fccb5a18-4f56-469a-a72e-7ab48f0dbd22',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-17T13:38:44.633189'},\n",
+      "                {   'created_at': '2024-12-15T17:44:37.975300',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': 'edb7fc33-ccb7-4bc4-8c5c-4e3d8b648ddb',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-15T17:44:40.490311'},\n",
+      "                {   'created_at': '2024-12-17T10:32:43.953229',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': '002413ca-e8ce-4baf-946b-c78b0d861ad3',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-17T10:32:48.022813'},\n",
+      "                {   'created_at': '2024-12-16T13:27:44.787943',\n",
+      "                    'detail': 'Finished',\n",
+      "                    'identifier': 'afbec9b5-7e46-4251-8e71-ec38479dbb11',\n",
+      "                    'progress': 100.0,\n",
+      "                    'status': 'FINISHED',\n",
+      "                    'updated_at': '2024-12-16T13:27:46.640572'}],\n",
+      "    'numberMatched': 18}\n",
+      "Deleting all the jobs...\n",
+      "{'jobs': [], 'numberMatched': 0}\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "post_response = session.get(f\"{href_staging}/jobs\",                               \n",
+    "                              timeout = TIMEOUT,)\n",
+    "\n",
+    "jobs = json.loads(post_response.content)\n",
+    "pprint.PrettyPrinter(indent=4).pprint(jobs)\n",
+    "if jobs.get(\"numberMatched\") > 0:\n",
```

Comment:
Here we see that it is not user-friendly if the wrapped function would return a dict that the end-user has to manipulate. Maybe the wrapped function could return directly the number of matched jobs. In the demo you would do something like: 
```py
jobs = staging_client.get_jobs()
if len(jobs) > 0: 
    # ...
```